src/components/new-print-page/new-print-page-2.jsx

```diff
@@ -50,33 +47,36 @@ const NewPrintPage2 = () => {
   return (
     <div>
       <tableContext.Provider value={{ table, setTable }}>
-        <p>inf is {JSON.stringify(info)}</p>
-
         <Table Type="res" sensorsProbsNum={2} editable={true}></Table>
 
         <button
           onClick={() => {
-
+            history.push("/new-print");
+          }}
+        >
+          BACK
+        </button>
+        <button
+          onClick={() => {
             const processedTable = processData(table);
             fetch(`${endpointUrl}/print-resistance-table`, {
               method: "POST",
               body: JSON.stringify(processedTable),
               headers: {
                 "Content-Type": "application/json"
               }
-            });
-            setInfo(null);
-            history.push("/");
-          }}
-        >
-          FINISK
-        </button>
-        <button
-          onClick={() => {
-            history.push("/new-print");
+            })
+              .then(() => {
+                setInfo(null);
+                history.push("/");
+              })
+              .catch(err => {
+                setInfo(null);
+                history.push("/");
```

Comment:
It would be better here to display an error message here, rather than `history.push('/')`. You want to make it clear to users that something has gone wrong